t/56_get_followers.t

```diff
@@ -0,0 +1,100 @@
+use strict;
+use warnings;
```

Comment:
nits: By using `Test2::V0`, `strict` and `warnings` are also imported to gether. These two lines does not have to be here.